lib/core/src/Cardano/Pool/DB/Log.hs

```diff
@@ -0,0 +1,67 @@
+{-# LANGUAGE LambdaCase #-}
+
+-- |
+-- Copyright: © 2018-2020 IOHK
+-- License: Apache-2.0
+--
+-- Logging types specific to the pool database.
+--
+module Cardano.Pool.DB.Log
+    ( PoolDbLog (..)
+    ) where
+
+import Prelude
+
+import Cardano.BM.Data.Severity
+    ( Severity (..) )
+import Cardano.BM.Data.Tracer
+    ( HasPrivacyAnnotation (..), HasSeverityAnnotation (..) )
+import Cardano.DB.Sqlite
+    ( DBLog (..) )
+import Cardano.Wallet.Logging
+    ( BracketLog )
+import Cardano.Wallet.Primitive.Types
+    ( EpochNo, PoolId, PoolRetirementCertificate )
+import Data.Text.Class
+    ( ToText (..), toText )
+import Fmt
+    ( pretty )
+
+import qualified Data.Text as T
+
+data PoolDbLog
+    = MsgGeneric DBLog
+    | MsgRemovingPool PoolId
+    | MsgRemovingRetiredPools [PoolRetirementCertificate]
+    | MsgRemovingRetiredPoolsForEpoch EpochNo BracketLog
+    deriving (Eq, Show)
+
+instance HasPrivacyAnnotation PoolDbLog
+
+instance HasSeverityAnnotation PoolDbLog where
+    getSeverityAnnotation = \case
+        MsgGeneric e -> getSeverityAnnotation e
+        MsgRemovingPool {} -> Notice
+        MsgRemovingRetiredPools {} -> Notice
+        MsgRemovingRetiredPoolsForEpoch {} -> Notice
```

Comment:
I don't think all of these should be `Notice` level. Ideally there should be one notice for when a pool is registered and just one more notice for when it retires. The rest is probably debug.

Reply:
> I don't think all of these should be Notice level. Ideally there should be one notice for when a pool is registered and just one more notice for when it retires. The rest is probably debug.

Fixed in 024d36b7c51dac6cb9624697c88fcfa32b82f997.